packages/x-charts/src/internals/plugins/featurePlugins/useChartCartesianAxis/useChartCartesianAxisRendering.selectors.ts

```diff
@@ -264,3 +271,77 @@ export const selectorChartRawAxis = createSelector(
     return axis;
   },
 );
+
+export const selectorChartDefaultXAxisId = createSelector(
+  [selectorChartRawXAxis],
+  (xAxes) => xAxes![0].id,
+);
```

Comment:
why a selector for the default axis id? 🤔

Reply:
I think my initial thought process was to avoid recomputing `selectorChartSeriesFlatbushMap` when the axes changed, but a change in the axes is causing a recomputation anyway because of the dependency on `selectorChartXScales` and `selectorChartYScales`, so I don't think there's a big benefit. 

If you want to, I can remove it, but I don't think it hurts either

Reply:
Feels a bit odd, but not a problem either